pt/lessons/libraries/benchee.md

```diff
@@ -0,0 +1,333 @@
+---
+version: 1.0.1
+title: Benchee
+redirect_from:
+  - /lessons/libraries/benchee/
+---
+
+Não podemos simplesmente adivinhar quais funções são rápidas e que são lentas - precisamos de medidas reais quando estamos curiosos. É aí que _benchmarking_ entra. Nesta lição, aprenderemos sobre como é fácil medir a velocidade do nosso código.
+
+{% include toc.html %}
+
+# Sobre Benchee
+
+Enquanto existe uma [função no Erlang](http://erlang.org/doc/man/timer.html#tc-1) que pode ser usada para medição muito básica do tempo de execução de uma função, ela não é tão boa de usar como algumas das ferramentas disponíveis e não lhe dá várias medidas para obter boas estatísticas, então vamos usar [Benchee](https://github.com/PragTob/benchee). Benchee nos fornece uma série de estatísticas com comparações fáceis entre cenários, uma ótima característica que nos permite testar diferentes entradas para as funções que estamos avaliando, e vários formatadores diferentes que podemos usar para mostrar nossos resultados, assim como a capacidade de escrever seu próprio formatador se desejado.
+
+# Uso 
+
+Para adicionar Benchee ao seu projeto, adicione-o como uma dependência ao seu arquivo `mix.exs`:
+```elixir
+defp deps do
+  [{:benchee, "~> 0.9", only: :dev}]
+end
+```
+Então chamamos:
+
+```shell
+$ mix deps.get
+...
+$ mix compile
+```
+
+O primeiro comando vai baixar e instalar o Benchee. Você pode ser solicitado a instalar o Hex junto com ele. O segundo compila a aplicação Benchee. Agora estamos prontos para escrever nosso primeiro _benchmark_!
+
+**Uma nota importante antes de começarmos:** Quando avaliar comparativamente, é muito importante não usar `iex` uma vez que isso funciona de forma diferente e é frequentemente muito mais lento do que seu código provavelmente usado em produção. Então, vamos criar um arquivo que chamaremos `benchmark.exs`, e nesse arquivo vamos adicionar o seguinte código. 
```

Comment:
I feel the word "provavelmente" could be removed, it looks odd in the phrase. I think the idea of the phrase will not change in this case.